### PR TITLE
Auth page

### LINKED ISSUE #51: UI Design for Login page
**Here is the final design. Please refer to screenshot below.**

evalai_redesign_loginpage_final

You can refer to the following image for further details on creating the page : 
evalai_redesign_loginpage_final_documentation

You can get all the related assets for creating the page from the link below : 
https://drive.google.com/drive/folders/1JsK6l1B5pSKJY2MCSMQVZYq1EwGuOx4t

Please comment below if you want to work on this or in case of any question/suggestion.

Mentors for EvalAI-ngx Project (**@aka-jain @spyshiv @shivaniprakash95 @RishabhJain2018 @deshraj**) will try to respond to this thread ASAP so that you are not blocked.

### PR DESCRIPTION
**Pull request by live-wire**
This is supposed to close #51 
Contains changes for login and signup page at path `auth/login` and `auth/signup`

Depends on #70 

Please have a look @aka-jain @spyshiv @RishabhJain2018 @shivaniprakash95

### CODE REVIEW

File: src/app/auth/auth.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="auth-container" style="background-image: url('assets/images/auth_background.png');">
+    <div class="auth-background-image" style="background-image: url('assets/images/auth_orange_image.png');">
+      <span class="auth-tagline">Evaluating state of the art in AI</span>
+    </div>
+    <div class="auth-form">
+      <div class="auth-form-logo">
+        <img src="assets/images/evalai_logo_light.png" />
+      </div>
+      <div class="auth-nav">
+        <a [class.auth-selected]="localRouter.url=='/auth/login'"  (click)="navigateTo('/auth/login')">LOGIN</a>
```
Comment (aka-jain) [inline src/app/auth/auth.component.html]: Adding click to anchor tag is not a good Practice instead use routerLink for the navigation @live-wire

Comment (aka-jain) [reply src/app/auth/auth.component.html]: Use like this `<a [routerLink]="['URL']" routerLinkActive="auth-selected">` @live-wire

File: src/app/auth/auth.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <a [class.auth-selected]="localRouter.url=='/auth/signup'" (click)="navigateTo('/auth/signup')">SIGNUP</a>
+      </div>
+      <router-outlet></router-outlet>
+      <div class="auth-form-footer">
+        <table>
```
Comment (aka-jain) [inline src/app/auth/auth.component.html]: Don't use table, instead use `ul li` elements they are easily customisable. @live-wire

File: src/app/input/input.component.scss [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    color: white;
+    border-bottom: 1px solid $gray-medium;
+    background: transparent;
+    padding-bottom:20px;
+    font-weight:$fw-light;
+    &:focus {
+      outline: none;
+    }
+  }
+  .input-field.theme-dark{
```
Comment (aka-jain) [inline src/app/input/input.component.scss]: You can write this as 
`.input-field{
     &.theme-dark{
     }
}`
 also @live-wire

File: src/app/signup/signup.component.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    		cursor:pointer;
+    	}
+  }
+}
+@include screen-small {
+.login-container {
+  .auth-forgot-div {
+  .auth-forgot {
+	width:150px;
+	}}}
```
Comment (aka-jain) [inline src/app/signup/signup.component.scss]: Indentations @live-wire

**Review by aka-jain — CHANGES_REQUESTED** [review]

**Comment by aka-jain** [comment]
Attach screenshot of your final result @live-wire

**Comment by live-wire** [comment]
auth
This is how the page looks like ☝️ 
> This PR is dependent on #70 (Please merge that before merging this)



I made the changes that you suggested @aka-jain . Please have a look 🥘 

cc: @RishabhJain2018 @spyshiv

**Comment by RishabhJain2018** [comment]
I have tested it. It works fine as of now but it is a static page, I think we should integrate it backend.

**Comment by RishabhJain2018** [comment]
@live-wire Please resolve the merge conflicts.

**Comment by RishabhJain2018** [comment]
@live-wire If this is completed then please move it to under review section, so that we can check it.

**Comment by RishabhJain2018** [comment]
@live-wire I tried to test it on my local, but it is giving me some error
While logging in the error screenshot is
screen shot 2018-06-26 at 1 51 09 pm
Also, while signing up the error screenshot is:
screen shot 2018-06-26 at 1 52 18 pm

**Comment by live-wire** [comment]
@RishabhJain2018 I moved the form error handling logic to global service! 
I think there is no use displaying all server-error responses to the user. 
I decided if the error is internal like above, I'd display this in the toast: 
`something went wrong <code>`

Please try now. 
I am still facing an issue while signing up. Signing up seems to create an account successfully but the API response is a 500 with the following error:
```
TypeError at /api/auth/registration/
is_safe_url() got an unexpected keyword argument 'allowed_hosts'

Request Method: POST
Request URL: http://localhost:8000/api/auth/registration/
Django Version: 1.10.2
Python Executable: /usr/local/bin/python
Python Version: 2.7.14
Python Path: ['/code', '/usr/local/lib/python27.zip', '/usr/local/lib/python2.7', '/usr/local/lib/python2.7/plat-linux2', '/usr/local/lib/python2.7/lib-tk', '/usr/local/lib/python2.7/lib-old', '/usr/local/lib/python2.7/lib-dynload', '/usr/local/lib/python2.7/site-packages', '/code/apps', '/usr/local/lib/python2.7/site-packages/odf', '/usr/local/lib/python2.7/site-packages/odf', '/usr/local/lib/python2.7/site-packages/odf', '/usr/local/lib/python2.7/site-packages/odf', '/usr/local/lib/python2.7/site-packages/odf', '/usr/local/lib/python2.7/site-packages/odf', '/usr/local/lib/python2.7/site-packages/odf']
Server time: Wed, 27 Jun 2018 23:18:59 +0000
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.sites',
 'accounts',
 'analytics',
 'base',
 'challenges',
 'hosts',
 'jobs',
 'participants',
 'web',
 'allauth',
 'allauth.account',
 'corsheaders',
 'django_ses',
 'import_export',
 'rest_auth',
 'rest_auth.registration',
 'rest_framework.authtoken',
 'rest_framework',
 'rest_framework_docs',
 'rest_framework_expiring_authtoken',
 'django_spaghetti',
 'autofixture',
 'debug_toolbar',
 'django_extensions',
 'silk']
Installed Middleware:
['corsheaders.middleware.CorsMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'debug_toolbar.middleware.DebugToolbarMiddleware',
 'silk.middleware.SilkyMiddleware']


Traceback:  

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/exception.py" in inner
  39.             response = get_response(request)

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python2.7/site-packages/django/views/decorators/csrf.py" in wrapped_view
  58.         return view_func(*args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapper
  67.             return bound_func(*args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/django/views/decorators/debug.py" in sensitive_post_parameters_wrapper
  76.             return view(request, *args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/django/utils/decorators.py" in bound_func
  63.                 return func.__get__(self, type(self))(*args2, **kwargs2)

File "/usr/local/lib/python2.7/site-packages/rest_auth/registration/views.py" in dispatch
  39.         return super(RegisterView, self).dispatch(*args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py" in dispatch
  477.             response = self.handle_exception(exc)

File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py" in handle_exception
  437.             self.raise_uncaught_exception(exc)

File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py" in dispatch
  474.             response = handler(request, *args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/rest_framework/generics.py" in post
  191.         return self.create(request, *args, **kwargs)

File "/usr/local/lib/python2.7/site-packages/rest_auth/registration/views.py" in create
  58.         user = self.perform_create(serializer)

File "/usr/local/lib/python2.7/site-packages/rest_auth/registration/views.py" in perform_create
  74.                         None)

File "/usr/local/lib/python2.7/site-packages/allauth/account/utils.py" in complete_signup
  183.                          signal_kwargs=signal_kwargs)

File "/usr/local/lib/python2.7/site-packages/allauth/account/utils.py" in perform_login
  152.             get_login_redirect_url(request, redirect_url))

File "/usr/local/lib/python2.7/site-packages/allauth/account/utils.py" in get_login_redirect_url
  50.             redirect_field_name=redirect_field_name) or

File "/usr/local/lib/python2.7/site-packages/allauth/account/utils.py" in get_next_redirect_url
  36.     if not get_adapter(request).is_safe_url(redirect_to):

File "/usr/local/lib/python2.7/site-packages/allauth/account/adapter.py" in is_safe_url
  410.         return is_safe_url(url, allowed_hosts=None)

Exception Type: TypeError at /api/auth/registration/
Exception Value: is_safe_url() got an unexpected keyword argument 'allowed_hosts'
Request information:
USER: dbadbadba

GET: No GET data

POST: No POST data

FILES: No FILES data

COOKIES: No cookie data

META:
CONTENT_LENGTH = '105'
CONTENT_TYPE = 'application/json'
CSRF_COOKIE = u'SG2gyXyeOkVk6aRtOfSveBSj2963eEeuN6zcfGmX51FSfFCCIR978gP1uTFgIYrs'
CSRF_COOKIE_USED = True
DEBUG = 'True'
DJANGO_SERVER = 'django'
DJANGO_SETTINGS_MODULE = 'settings'
GATEWAY_INTERFACE = 'CGI/1.1'
GPG_KEY = 'C01E1CAD5EA2C4F0B8E3571504C367C218ADD4FF'
HOME = '/root'
HOSTNAME = 'django'
HTTP_ACCEPT = 'application/json, text/plain, */*'
HTTP_ACCEPT_ENCODING = 'gzip, deflate, br'
HTTP_ACCEPT_LANGUAGE = 'en-US,en-NL;q=0.9,en;q=0.8'
HTTP_CACHE_CONTROL = 'no-cache'
HTTP_CONNECTION = 'keep-alive'
HTTP_HOST = 'localhost:8000'
HTTP_ORIGIN = 'http://localhost:4200'
HTTP_PRAGMA = 'no-cache'
HTTP_REFERER = 'http://localhost:4200/auth/signup'
HTTP_USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 Safari/537.36'
LANG = 'C.UTF-8'
OLDPWD = '/code'
PATH = '/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
PATH_INFO = u'/api/auth/registration/'
POSTGRES_HOST = 'db'
POSTGRES_NAME = 'postgres'
POSTGRES_PASSWORD = 'postgres'
POSTGRES_PORT = '5432'
POSTGRES_USER = 'postgres'
PWD = '/code'
PYTHONUNBUFFERED = '1'
PYTHON_PIP_VERSION = '9.0.1'
PYTHON_VERSION = '2.7.14'
QUERY_STRING = ''
RABBITMQ_HOST = 'rabbitmq'
REMOTE_ADDR = '172.20.0.1'
REMOTE_HOST = ''
REQUEST_METHOD = 'POST'
RUN_MAIN = 'true'
SCRIPT_NAME = u''
SERVER_NAME = 'django'
SERVER_PORT = '8000'
SERVER_PROTOCOL = 'HTTP/1.1'
SERVER_SOFTWARE = 'WSGIServer/0.1 Python/2.7.14'
SHLVL = '0'
TZ = 'UTC'
wsgi.errors = <open file '<stderr>', mode 'w' at 0x7f45967311e0>
wsgi.file_wrapper = ''
wsgi.input = <socket._fileobject object at 0x7f4582b05ad0>
wsgi.multiprocess = False
wsgi.multithread = True
wsgi.run_once = False
wsgi.url_scheme = 'http'
wsgi.version = 

Settings:
Using settings module settings
ABSOLUTE_URL_OVERRIDES = {}
ACCOUNT_CONFIRM_EMAIL_ON_GET = True
ACCOUNT_DEFAULT_HTTP_PROTOCOL = 'http'
ACCOUNT_EMAIL_CONFIRMATION_ANONYMOUS_REDIRECT_URL = '/api/auth/email-confirmed/'
ACCOUNT_EMAIL_CONFIRMATION_AUTHENTICATED_REDIRECT_URL = '/api/auth/email-confirmed/'
ACCOUNT_EMAIL_REQUIRED = True
ADMINS = []
ADMIN_EMAIL = 'admin@cloudcv.org'
ALLOWED_HOSTS = []
APPEND_SLASH = True
APPS_DIR = '/code/apps'
AUTHENTICATION_BACKENDS = 
AUTH_PASSWORD_VALIDATORS = u'********************'
AUTH_USER_MODEL = u'auth.User'
BASE_DIR = '/code'
CACHES = {'default': {'BACKEND': 'django.core.cache.backends.dummy.DummyCache'}}
CACHE_MIDDLEWARE_ALIAS = u'default'
CACHE_MIDDLEWARE_KEY_PREFIX = u'********************'
CACHE_MIDDLEWARE_SECONDS = 600
CLOUDCV_TEAM_EMAIL = 'EvalAI Team <team@cloudcv.org>'
CORS_ORIGIN_ALLOW_ALL = True
CSRF_COOKIE_AGE = 31449600
CSRF_COOKIE_DOMAIN = None
CSRF_COOKIE_HTTPONLY = False
CSRF_COOKIE_NAME = u'csrftoken'
CSRF_COOKIE_PATH = u'/'
CSRF_COOKIE_SECURE = False
CSRF_FAILURE_VIEW = u'django.views.csrf.csrf_failure'
CSRF_HEADER_NAME = u'HTTP_X_CSRFTOKEN'
CSRF_TRUSTED_ORIGINS = []
DATABASES = {'default': {'ENGINE': 'django.db.backends.postgresql_psycopg2', 'AUTOCOMMIT': True, 'ATOMIC_REQUESTS': False, 'NAME': 'postgres', 'CONN_MAX_AGE': 0, 'TIME_ZONE': None, 'PORT': '5432', 'HOST': 'db', 'USER': 'postgres', 'TEST': {'COLLATION': None, 'CHARSET': None, 'NAME': None, 'MIRROR': None}, 'PASSWORD': u'********************', 'OPTIONS': {}}}
DATABASE_ROUTERS = []
DATA_UPLOAD_MAX_MEMORY_SIZE = 524288000
DATA_UPLOAD_MAX_NUMBER_FIELDS = 1000
DATETIME_FORMAT = u'N j, Y, P'
DATETIME_INPUT_FORMATS = [u'%Y-%m-%d %H:%M:%S', u'%Y-%m-%d %H:%M:%S.%f', u'%Y-%m-%d %H:%M', u'%Y-%m-%d', u'%m/%d/%Y %H:%M:%S', u'%m/%d/%Y %H:%M:%S.%f', u'%m/%d/%Y %H:%M', u'%m/%d/%Y', u'%m/%d/%y %H:%M:%S', u'%m/%d/%y %H:%M:%S.%f', u'%m/%d/%y %H:%M', u'%m/%d/%y']
DATE_FORMAT = u'N j, Y'
DATE_INPUT_FORMATS = [u'%Y-%m-%d', u'%m/%d/%Y', u'%m/%d/%y', u'%b %d %Y', u'%b %d, %Y', u'%d %b %Y', u'%d %b, %Y', u'%B %d %Y', u'%B %d, %Y', u'%d %B %Y', u'%d %B, %Y']
DEBUG = True
DEBUG_PROPAGATE_EXCEPTIONS = False
DECIMAL_SEPARATOR = u'.'
DEFAULT_APPS = ['django.contrib.admin', 'django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles', 'django.contrib.sites']
DEFAULT_CHARSET = u'utf-8'
DEFAULT_CONTENT_TYPE = u'text/html'
DEFAULT_EXCEPTION_REPORTER_FILTER = u'django.views.debug.SafeExceptionReporterFilter'
DEFAULT_FILE_STORAGE = u'django.core.files.storage.FileSystemStorage'
DEFAULT_FROM_EMAIL = u'webmaster@localhost'
DEFAULT_INDEX_TABLESPACE = u''
DEFAULT_TABLESPACE = u''
DISALLOWED_USER_AGENTS = []
EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
EMAIL_HOST = 'localhost'
EMAIL_HOST_PASSWORD = u'********************'
EMAIL_HOST_USER = u''
EMAIL_PORT = 1025
EMAIL_SSL_CERTFILE = None
EMAIL_SSL_KEYFILE = u'********************'
EMAIL_SUBJECT_PREFIX = u'[Django] '
EMAIL_TIMEOUT = None
EMAIL_USE_SSL = False
EMAIL_USE_TLS = False
EXPIRING_TOKEN_LIFESPAN = u'********************'
FILE_CHARSET = u'utf-8'
FILE_UPLOAD_DIRECTORY_PERMISSIONS = None
FILE_UPLOAD_HANDLERS = [u'django.core.files.uploadhandler.MemoryFileUploadHandler', u'django.core.files.uploadhandler.TemporaryFileUploadHandler']
FILE_UPLOAD_MAX_MEMORY_SIZE = 524288000
FILE_UPLOAD_PERMISSIONS = None
FILE_UPLOAD_TEMP_DIR = None
FIRST_DAY_OF_WEEK = 0
FIXTURE_DIRS = []
FORCE_SCRIPT_NAME = None
FORMAT_MODULE_PATH = None
IGNORABLE_404_URLS = []
INSTALLED_APPS = ['django.contrib.admin', 'django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles', 'django.contrib.sites', 'accounts', 'analytics', 'base', 'challenges', 'hosts', 'jobs', 'participants', 'web', 'allauth', 'allauth.account', 'corsheaders', 'django_ses', 'import_export', 'rest_auth', 'rest_auth.registration', 'rest_framework.authtoken', 'rest_framework', 'rest_framework_docs', 'rest_framework_expiring_authtoken', 'django_spaghetti', 'autofixture', 'debug_toolbar', 'django_extensions', 'silk']
INTERNAL_IPS = []
LANGUAGES = [(u'af', u'Afrikaans'), (u'ar', u'Arabic'), (u'ast', u'Asturian'), (u'az', u'Azerbaijani'), (u'bg', u'Bulgarian'), (u'be', u'Belarusian'), (u'bn', u'Bengali'), (u'br', u'Breton'), (u'bs', u'Bosnian'), (u'ca', u'Catalan'), (u'cs', u'Czech'), (u'cy', u'Welsh'), (u'da', u'Danish'), (u'de', u'German'), (u'dsb', u'Lower Sorbian'), (u'el', u'Greek'), (u'en', u'English'), (u'en-au', u'Australian English'), (u'en-gb', u'British English'), (u'eo', u'Esperanto'), (u'es', u'Spanish'), (u'es-ar', u'Argentinian Spanish'), (u'es-co', u'Colombian Spanish'), (u'es-mx', u'Mexican Spanish'), (u'es-ni', u'Nicaraguan Spanish'), (u'es-ve', u'Venezuelan Spanish'), (u'et', u'Estonian'), (u'eu', u'Basque'), (u'fa', u'Persian'), (u'fi', u'Finnish'), (u'fr', u'French'), (u'fy', u'Frisian'), (u'ga', u'Irish'), (u'gd', u'Scottish Gaelic'), (u'gl', u'Galician'), (u'he', u'Hebrew'), (u'hi', u'Hindi'), (u'hr', u'Croatian'), (u'hsb', u'Upper Sorbian'), (u'hu', u'Hungarian'), (u'ia', u'Interlingua'), (u'id', u'Indonesian'), (u'io', u'Ido'), (u'is', u'Icelandic'), (u'it', u'Italian'), (u'ja', u'Japanese'), (u'ka', u'Georgian'), (u'kk', u'Kazakh'), (u'km', u'Khmer'), (u'kn', u'Kannada'), (u'ko', u'Korean'), (u'lb', u'Luxembourgish'), (u'lt', u'Lithuanian'), (u'lv', u'Latvian'), (u'mk', u'Macedonian'), (u'ml', u'Malayalam'), (u'mn', u'Mongolian'), (u'mr', u'Marathi'), (u'my', u'Burmese'), (u'nb', u'Norwegian Bokm\xe5l'), (u'ne', u'Nepali'), (u'nl', u'Dutch'), (u'nn', u'Norwegian Nynorsk'), (u'os', u'Ossetic'), (u'pa', u'Punjabi'), (u'pl', u'Polish'), (u'pt', u'Portuguese'), (u'pt-br', u'Brazilian Portuguese'), (u'ro', u'Romanian'), (u'ru', u'Russian'), (u'sk', u'Slovak'), (u'sl', u'Slovenian'), (u'sq', u'Albanian'), (u'sr', u'Serbian'), (u'sr-latn', u'Serbian Latin'), (u'sv', u'Swedish'), (u'sw', u'Swahili'), (u'ta', u'Tamil'), (u'te', u'Telugu'), (u'th', u'Thai'), (u'tr', u'Turkish'), (u'tt', u'Tatar'), (u'udm', u'Udmurt'), (u'uk', u'Ukrainian'), (u'ur', u'Urdu'), (u'vi', u'Vietnamese'), (u'zh-hans', u'Simplified Chinese'), (u'zh-hant', u'Traditional Chinese')]
LANGUAGES_BIDI = [u'he', u'ar', u'fa', u'ur']
LANGUAGE_CODE = 'en-us'
LANGUAGE_COOKIE_AGE = None
LANGUAGE_COOKIE_DOMAIN = None
LANGUAGE_COOKIE_NAME = u'django_language'
LANGUAGE_COOKIE_PATH = u'/'
LOCALE_PATHS = []
LOGGING = {'loggers': {'django.security': {'handlers': ['mail_admins'], 'propagate': False, 'level': 'ERROR'}, 'django.request': {'handlers': ['mail_admins'], 'propagate': False, 'level': 'ERROR'}, 'django.db.backends': {'handlers': ['mail_admins'], 'propagate': False, 'level': 'ERROR'}, 'django': {'propagate': False, 'handlers': ['console']}}, 'disable_existing_loggers': False, 'handlers': {'logfile': {'backupCount': 10, 'level': 'DEBUG', 'filename': '/code/django.log', 'formatter': 'verbose', 'class': 'logging.handlers.RotatingFileHandler', 'maxBytes': 50000}, 'console': {'formatter': 'simple', 'class': 'logging.StreamHandler', 'filters': ['require_debug_true'], 'level': 'INFO'}, 'mail_admins': {'class': 'django.utils.log.AdminEmailHandler', 'filters': ['require_debug_false'], 'level': 'ERROR'}}, 'formatters': {'simple': {'datefmt': '%Y-%m-%d %H:%M:%S', 'format': '[%(asctime)s] %(levelname)s %(message)s'}, 'verbose': {'datefmt': '%Y-%m-%d %H:%M:%S', 'format': '[%(asctime)s] %(levelname)s %(module)s %(message)s'}}, 'version': 1, 'filters': {'require_debug_false': {'()': 'django.utils.log.RequireDebugFalse'}, 'require_debug_true': {'()': 'django.utils.log.RequireDebugTrue'}}, 'root': {'level': 'INFO', 'handlers': ['console']}}
LOGGING_CONFIG = u'logging.config.dictConfig'
LOGIN_REDIRECT_URL = u'/accounts/profile/'
LOGIN_URL = u'/accounts/login/'
LOGOUT_REDIRECT_URL = None
MANAGERS = []
MEDIA_ROOT = '/code/media'
MEDIA_URL = '/media/'
MESSAGE_STORAGE = u'django.contrib.messages.storage.fallback.FallbackStorage'
MIDDLEWARE = ['corsheaders.middleware.CorsMiddleware', 'django.middleware.security.SecurityMiddleware', 'django.contrib.sessions.middleware.SessionMiddleware', 'django.middleware.common.CommonMiddleware', 'django.middleware.csrf.CsrfViewMiddleware', 'django.contrib.auth.middleware.AuthenticationMiddleware', 'django.contrib.messages.middleware.MessageMiddleware', 'django.middleware.clickjacking.XFrameOptionsMiddleware', 'debug_toolbar.middleware.DebugToolbarMiddleware', 'silk.middleware.SilkyMiddleware']
MIDDLEWARE_CLASSES = [u'django.middleware.common.CommonMiddleware', u'django.middleware.csrf.CsrfViewMiddleware']
MIGRATION_MODULES = {}
MONTH_DAY_FORMAT = u'F j'
NUMBER_GROUPING = 0
OLD_PASSWORD_FIELD_ENABLED = u'********************'
OUR_APPS = ['accounts', 'analytics', 'base', 'challenges', 'hosts', 'jobs', 'participants', 'web']
PASSWORD_HASHERS = u'********************'
PASSWORD_RESET_TIMEOUT_DAYS = u'********************'
PREPEND_WWW = False
RABBITMQ_PARAMETERS = {'HOST': 'rabbitmq', 'EVALAI_EXCHANGE': {'TYPE': 'topic', 'NAME': 'evalai_submissions'}, 'SUBMISSION_QUEUE': 'submission_task_queue'}
REST_AUTH_SERIALIZERS = {'USER_DETAILS_SERIALIZER': 'accounts.serializers.ProfileSerializer'}
REST_FRAMEWORK = {'DEFAULT_AUTHENTICATION_CLASSES': ['rest_framework_expiring_authtoken.authentication.ExpiringTokenAuthentication'], 'DEFAULT_PERMISSION_CLASSES': ['rest_framework.permissions.IsAuthenticatedOrReadOnly'], 'PAGE_SIZE': 10, 'DEFAULT_THROTTLE_RATES': {'anon': '100/minute', 'user': '100/minute'}, 'DEFAULT_RENDERER_CLASSES': ('rest_framework.renderers.JSONRenderer',), 'DEFAULT_THROTTLE_CLASSES': ('rest_framework.throttling.AnonRateThrottle', 'rest_framework.throttling.UserRateThrottle'), 'TEST_REQUEST_DEFAULT_FORMAT': 'json', 'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination'}
ROOT_URLCONF = 'evalai.urls'
SECRET_KEY = u'********************'
SECURE_BROWSER_XSS_FILTER = False
SECURE_CONTENT_TYPE_NOSNIFF = False
SECURE_HSTS_INCLUDE_SUBDOMAINS = False
SECURE_HSTS_SECONDS = 0
SECURE_PROXY_SSL_HEADER = None
SECURE_REDIRECT_EXEMPT = []
SECURE_SSL_HOST = None
SECURE_SSL_REDIRECT = False
SERVER_EMAIL = u'root@localhost'
SESSION_CACHE_ALIAS = u'default'
SESSION_COOKIE_AGE = 1209600
SESSION_COOKIE_DOMAIN = None
SESSION_COOKIE_HTTPONLY = True
SESSION_COOKIE_NAME = u'sessionid'
SESSION_COOKIE_PATH = u'/'
SESSION_COOKIE_SECURE = False
SESSION_ENGINE = u'django.contrib.sessions.backends.db'
SESSION_EXPIRE_AT_BROWSER_CLOSE = False
SESSION_FILE_PATH = None
SESSION_SAVE_EVERY_REQUEST = False
SESSION_SERIALIZER = u'django.contrib.sessions.serializers.JSONSerializer'
SETTINGS_MODULE = 'settings'
SHORT_DATETIME_FORMAT = u'm/d/Y P'
SHORT_DATE_FORMAT = u'm/d/Y'
SIGNING_BACKEND = u'django.core.signing.TimestampSigner'
SILENCED_SYSTEM_CHECKS = []
SILKY_PYTHON_PROFILER = True
SITE_ID = 1
SPAGHETTI_SAUCE = {'apps': ['auth', 'accounts', 'analytics', 'base', 'challenges', 'hosts', 'jobs', 'participants', 'web'], 'show_fields': True}
STATICFILES_DIRS = []
STATICFILES_FINDERS = [u'django.contrib.staticfiles.finders.FileSystemFinder', u'django.contrib.staticfiles.finders.AppDirectoriesFinder']
STATICFILES_STORAGE = u'django.contrib.staticfiles.storage.StaticFilesStorage'
STATIC_ROOT = '/code/static'
STATIC_URL = '/static/'
TEMPLATES = [{'DIRS': [], 'APP_DIRS': True, 'OPTIONS': {'context_processors': ['django.template.context_processors.debug', 'django.template.context_processors.request', 'django.contrib.auth.context_processors.auth', 'django.contrib.messages.context_processors.messages']}, 'BACKEND': 'django.template.backends.django.DjangoTemplates'}]
TEST = []
TEST_NON_SERIALIZED_APPS = []
TEST_RUNNER = u'django.test.runner.DiscoverRunner'
THIRD_PARTY_APPS = ['allauth', 'allauth.account', 'corsheaders', 'django_ses', 'import_export', 'rest_auth', 'rest_auth.registration', 'rest_framework.authtoken', 'rest_framework', 'rest_framework_docs', 'rest_framework_expiring_authtoken']
THOUSAND_SEPARATOR = u','
TIME_FORMAT = u'P'
TIME_INPUT_FORMATS = [u'%H:%M:%S', u'%H:%M:%S.%f', u'%H:%M']
TIME_ZONE = 'UTC'
USE_ETAGS = False
USE_I18N = True
USE_L10N = True
USE_THOUSAND_SEPARATOR = False
USE_TZ = True
USE_X_FORWARDED_HOST = False
USE_X_FORWARDED_PORT = False
WSGI_APPLICATION = 'evalai.wsgi.application'
X_FRAME_OPTIONS = u'SAMEORIGIN'
YEAR_MONTH_FORMAT = u'F Y'


You're seeing this error because you have DEBUG = True in your
Django settings file. Change that to False, and Django will
display a standard page generated by the handler for this status code.

```

**Comment by RishabhJain2018** [comment]
@live-wire Please resolve the merge conflicts.

**Comment by RishabhJain2018** [comment]
@live-wire I am able to signup but not login 😟 
Here is the screenshot of the error
screen shot 2018-07-06 at 4 33 21 pm

Moreover, please don't display error code in the notification. Also, notification can be a little more smoother.

**Comment by live-wire** [comment]
I fixed this login issue in one of the upcoming PRs 🙈 
Will fix it here as well. 
It is so difficult to maintain so many open PRs 😢

**Comment by live-wire** [comment]
I'll update this PR right after #66  and #71  are merged 💃

**Comment by RishabhJain2018** [comment]
closing this since it is already merged.

**Pull request closed without merging**